relation_engine/taxa/gtdb/loaders/gtdb_taxa_delta_loader.py

```diff
@@ -1,102 +1,55 @@
 from relation_engine.batchload.time_travelling_database import ArangoBatchTimeTravellingDB
 
+_INPUT_FILE = 'input_file'
 _LOAD_NAMESPACE = 'gtdb_taxa'
 
 
-def parse_args():
+def get_config():
     parser = argparse.ArgumentParser(description="""
 Load a GTDB taxonomy dump into an ArangoDB time travelling database, calculating and applying the
 changes between the prior load and the current load, and retaining the prior load.
 """.strip())
-    parser.add_argument('--file', required=True,
-                        help='the GTDB taxonomy file, e.g. bac120_taxonomy_r89.tsv')
-    parser.add_argument(
-        '--arango-url',
-        required=True,
-        help='The url of the ArangoDB server (e.g. http://localhost:8528')
-    parser.add_argument(
-        '--database',
-        required=True,
-        help='the name of the ArangoDB database that will be altered')
-    parser.add_argument(
-        '--user',
-        help='the ArangoDB user name; if --pwd-file is not included a password prompt will be ' +
-        'presented. Omit to connect with default credentials.')
-    parser.add_argument(
-        '--pwd-file',
-        help='the path to a file containing the ArangoDB password and nothing else; ' +
-        'if --user is included and --pwd-file is omitted a password prompt will be presented.')
-    parser.add_argument(
-        '--load-registry-collection',
-        required=True,
-        help='the name of the ArangoDB collection where the load will be registered. ' +
-        'This is typically the same collection for all delta loaded data.')
-    parser.add_argument(
-        '--node-collection',
-        required=True,
-        help='the name of the ArangoDB collection into which taxa nodes will be loaded')
-    parser.add_argument(
-        '--edge-collection',
-        required=True,
-        help='the name of the ArangoDB collection into which taxa edges will be loaded')
-    parser.add_argument(
-        '--load-version',
-        required=True,
-        help='the version of this load. This version will be added to a field in the nodes and ' +
-             'edges and will be used as part of the _key field.')
-    parser.add_argument(
-        '--load-timestamp',
-        type=int,
-        required=True,
-        help='the timestamp to be applied to the load, in unix epoch milliseconds. Any nodes ' +
-             'or edges created in this load will start to exist with this time stamp. ' +
-             'NOTE: the user is responsible for ensuring this timestamp is greater than any ' +
-             'other timestamps previously used to load data into the NCBI taxonomy DB.')
-    parser.add_argument(
-        '--release-timestamp',
-        type=int,
-        required=True,
-        help='the timestamp, in unix epoch milliseconds, when the data was released ' +
-        'at the source.')
-
-    return parser.parse_args()
+    parser.add_argument('--config', required=True,
```

Comment:
Basically the same reason you might use keyword arguments rather than positional arguments to a method - it makes it easier to extend later and maintain backwards compatibility. I've been bitten by this particular case before...

It also provides semantics for the argument.